datasets/saudinewsnet/README.MD

```diff
@@ -0,0 +1,139 @@
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Discussion of Social Impact and Biases](#discussion-of-social-impact-and-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** [SaudiNewsNet](https://github.com/parallelfold/SaudiNewsNet)
+- **Repository:** [Website](https://github.com/parallelfold/SaudiNewsNet)
+- **Paper:** [More Information Needed]
+- **Point of Contact:** [Mazen Abdulaziz](mailto:mazen.abdulaziz@gmail.com)
+
+### Dataset Summary
+
+The dataset contains a set of 2,972 Arabic newspaper articles alongwith metadata,
+extracted from various online Saudi newspapers and written in MSA. 
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+The dataset is based on Arabic.
+
+## Dataset Structure
+
+### Data Instances
+
+The dataset contains a set of 2,972 Arabic newspaper articles alongwith metadata, extracted from various online Saudi newspapers.
+
+### Data Fields
+
+```{'author': 'لندن: «الشرق الأوسط أونلاين»', 'content': 'أصدر الرئيس عبدربه منصور هادي رئيس الجمهورية اليمنية، اليوم (الاثنين)، قراراً جمهورياً بتعيين نايف البكري محافظا لعدن، وذلك بعد الانتصارات التي حققتها قوات المقاومة الشعبية في طرد المتمردين الحوثيين من المدينة.\n\n نايف البكري شغل منصب وكيل محافظة عدن لشؤون المديريات من قبل، وله علاقات جيدة مع مختلف القوى السياسية في المدينة، ويعتبر الرجل الأول.\n\n من جهة أخرى، أعلن بدر محمد باسملة وزير النقل اليمني عن إعداد خطة طوارئ للمدينة، مؤكدا أن أولوية الحكومة تشغيل الموانئ وتحسين الخدمات. موضحا ان فريقاً فنياً متخصصاً وصل من الإمارات لإعادة تشغيل المطار خلال أربع وعشرين ساعة واستئناف الرحلات من وإلى عدن خلال الساعات المقبلة. داعيا المنظمات الحقوقية والاعلاميين للقدومِ الى عدن لرصد المآسي التي ارتكبها المتمردون، مؤكداً ان الحكومة ستؤمن لهم رحلات من أي مطار في العالم.\n\n من جانبه، قال المتحدث الرسمي باسم الحكومة اليمنية راجح بادي، إن حجم الدمار في عدن لم يكن متوقعاً، لا سيما الذي طال البنى التحتية؛ فمطار عدن على سبيل المثال، مدمر بشكل شبه كامل، ومحطات توليد الكهرباء والمنشآت النفطية تعرضت هي الأخرى أكثر من مرة للقصف من قبل الحوثيين، حيث كانوا حينها يريدون الضغط على المقاومة الشعبية من خلال منعها من الحصول على المحروقات. وهو ما لم ينجحوا فيه. مبينا انه سيحتاج إعادة تأهيلها لمبالغ كبيرة وأشهر من العمل.', 'date_extracted': '2015-07-21 02:51:33', 'source': 'aawsat', 'title': 'الرئيس هادي يعين نايف البكري محافظا لعدن', 'url': 'http://aawsat.com/home/article/410801/الرئيس-هادي-يعين-نايف-البكري-محافظا-لعدن'}```
```

Comment:
thanks for providing an example from the dataset :) 

Can you move it to the `Data Instances` section instead ?

The `Data Fields` section is used to describe briefly each data field (author, content, ..).
Can you also add the data fields descriptions ?